### PR TITLE
Fifo animal shelter

### PR DESCRIPTION
**Pull request by daisyjanejohnson**
- [ x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x ] Picture of whiteboar

### CODE REVIEW

File: Data-Structures/fifoAnimalShelter/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Space O(1)
+The approach was to create two stacks and treat the values within them as though they were in a queue. Taking the first value in stack1 and pushing it into stack2 resulting in the FIFO method.
+## API
+enqueue(value) which inserts value into the PseudoQueue, using a first-in, first-out approach.
+dequeue() which extracts a value from the PseudoQueue, using a first-in, first-out approach.
+
+
+## WhiteBoard
+
+![WhiteBoard](.assets/AnimalShelter.png)
```
Comment (adrianhuebner) [inline Data-Structures/fifoAnimalShelter/README.md:21]: You aren't going up enough in the file path, change it to the following and it will work for you!

`../../assets/AnimalShelter.png`
